hera_mc/tests/test_weather.py

```diff
@@ -7,15 +7,15 @@
 """
 from __future__ import absolute_import, division, print_function
 
-import unittest
 import os
-import nose.tools as nt
 from math import floor
+
+import pytest
 import numpy as np
 from astropy.time import Time, TimeDelta
 
-from .. import mc, weather
-from . import TestHERAMC, is_onsite
+from .. import weather
+from . import onsite
 
 
 def test_reduce_time_vals():
```

Comment:
This would take a lot of thinking mostly because of how the expected values are calculated in the test too, but it could be possible to parametrize if desired.

Reply:
I'm not going to tackle this now, I'll add a comment.